modules/installation/pages/upgrade.adoc

```diff
@@ -199,28 +186,75 @@ Welcome to the TigerGraph platform installer!
 [PROGRESS]: 08:20:33 Switching to new version 4.2.1 successfully...
 ----
 
-=== Managing Service Disruption
+=== Managing Service Disruption During Rolling Upgrade
 
-During the rolling upgrade, expect a temporary service disruption. If the upgrade fails or the connection is lost, user can *resume* the upgrade process by running:
+During a rolling upgrade, temporary service interruptions are expected. If the upgrade process fails or the connection is unexpectedly lost, you can resume the upgrade by running:
```

Comment:
```suggestion
During a rolling upgrade, the TigerGraph service may be briefly disrupted, but it should resume on its on.
```
Comments: The developer didn't say it would resume on its own, but I'm guessing/hoping that is the case?
For very briefly disruptions, the TigerGraph system will often automatically "retry", so all of the requests that the user submitted still run; they just were delayed.  It would be helpful if the developer tells if if the user might need to resubmit some requests.  Also, I think it is better to tell the user to avoid long queries or loading jobs during an upgrade.  If you have to restart a long job, it's more of an inconvenience and lot of efficiency than restarting a short job.

Reply:
As for what do to **if the upgrade fails**, that is handled in another section and is more complicated. It is confusing/inconsistent/wrong to give simple instructions here.

As for what to do if "connection is unexpectedly lost", is it not clear what we mean by "connection". "Connection lost" is more of a symptom than a root cause.  "TigerGraph service disruption" could be considered a "lost connection", so the situations are overlapping.  In general, I think we should take this out here. Just address it properly and fully in the "Recovering from an Incomplete Rolling Backup"